### PR TITLE
Implemented paper trail for models

### PR DESCRIPTION
**Pull request by Spandana-Sai**
- Item, Person, Event
#10

### CODE REVIEW

File: config/initializers/paper_trail.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+PaperTrail.config.track_associations = false
```
Comment (kgf) [inline config/initializers/paper_trail.rb]: Please add the new line char. I have actually had issues when it was missing!
Thanks

**Comment by kgf** [comment]
One small change. Thanks for your help.

File: config/initializers/paper_trail.rb (hunk 1 above)
Comment (Spandana-Sai) [reply config/initializers/paper_trail.rb]: @kgf  done

**Comment by kgf** [comment]
That closes an old one. 
Thanks for your help.

**Pull request merged**